### DISCUSSION TITLE
Enhancement: Finish the alphabet file

### DISCUSSION THREAD

**Discussion by aflacc** [FunkinCrew/Funkin#1648] [Ideas]
So I was looking thru the issues (I do that when im bored) and i saw #852 which talks about the 8 in "EvilSk8r" missing but this is because the alphabet doesn't have all the characters working. I feel like you guys should make at least 0-9 work as well maybe even . , & ? and !

**Comment by haya3218** [comment]
other symbols are included in week 7 files, but other than . , ? and !, it doesn't have a corresponding letter anim

**Comment by ghost** [comment]
My Newgrounds name has numbers and it turns into #s
